### DISCUSSION TITLE
GeoFlutterFire2 Discussions!

### DISCUSSION THREAD

**Discussion by zeusbaba** [beerstorm-net/GeoFlutterFire2#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Varuhoord** [comment]
Hi, thank you for a great library. However, is there a possibility of using a future instead of a stream. I need to pull data only so often, not constantly as I see it drives up server writes and reads, especially when the user is moving. I've read the documentation, it says you can't use a limit on the call because it would put a limit on all 9 geohash calls, but what if there are 50,000 markers on a map and one user calls up the entire limit in one call, how would you go about solving that? Is there a way to construct a call to the server only so often and limit the data in a pagination sort of way, maybe using a timestamp? I'm experiencing a lot of problems trying to figure all this out.

**Reply by Varuhoord** [reply]
Can you please answer?

**Reply by Varuhoord** [reply]
Can you please answer?
